mod/dirfind.php

```diff
@@ -12,14 +12,13 @@
 use Friendica\Core\System;
 use Friendica\Core\Worker;
 use Friendica\Database\DBA;
-use Friendica\Model\Contact;
-use Friendica\Model\GContact;
+use Friendica\Model;
+use Friendica\Module\Contact;
```

Comment:
Although this is correct, please write `use Friendica\Module;` and later `Module\Contact::getContactTemplateVars` to prevent any immediate confusion.